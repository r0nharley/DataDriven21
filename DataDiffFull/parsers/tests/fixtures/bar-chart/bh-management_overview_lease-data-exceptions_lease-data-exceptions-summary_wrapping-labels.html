<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="982" height="431">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-o3d0swh-4">
            <rect x="0" y="0" width="385" height="639" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-o3d0swh-5">
            <rect x="0" y="0" width="385" height="639" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="982" height="431" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="333" y="10" width="639" height="385"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 333 87.5 L 972 87.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 333 164.5 L 972 164.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 333 241.5 L 972 241.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 333 318.5 L 972 318.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 333 395.5 L 972 395.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 333 10.5 L 972 10.5" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 332.5 10 L 332.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 616.5 10 L 616.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 900.5 10 L 900.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" opacity="1" class="highcharts-grid-line" d="M 403.5 10 L 403.5 395"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" opacity="1" class="highcharts-grid-line" d="M 474.5 10 L 474.5 395"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" opacity="1" class="highcharts-grid-line" d="M 545.5 10 L 545.5 395"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" opacity="1" class="highcharts-grid-line" d="M 687.5 10 L 687.5 395"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" opacity="1" class="highcharts-grid-line" d="M 758.5 10 L 758.5 395"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" opacity="1" class="highcharts-grid-line" d="M 829.5 10 L 829.5 395"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" opacity="1" class="highcharts-grid-line" d="M 971.5 10 L 971.5 395"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="333" y="10" width="639" height="385"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 333 87.5 L 323 87.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 333 164.5 L 323 164.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 333 241.5 L 323 241.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 333 318.5 L 323 318.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 333 395.5 L 323 395.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 333 9.5 L 323 9.5" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 332.5 10 L 332.5 395"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 333 395 L 972 395"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="url(https://staging.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-o3d0swh-5)">
            <rect x="360.5" y="45.5" width="8" height="594" fill="#6EDA55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-bar-series highcharts-color-undefined " transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-1 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="url(https://staging.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-o3d0swh-5)">
            <rect x="274.5" y="627.5" width="8" height="12" fill="#06e5ff" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-1 highcharts-bar-series highcharts-color-undefined " transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-2 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="url(https://staging.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-o3d0swh-5)">
            <rect x="188.5" y="605.5" width="8" height="34" fill="#fbb755" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-2 highcharts-bar-series highcharts-color-undefined " transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-3 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="url(https://staging.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-o3d0swh-5)">
            <rect x="102.5" y="639.5" width="8" height="0" fill="#218A8C" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-3 highcharts-bar-series highcharts-color-undefined " transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-4 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="url(https://staging.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-o3d0swh-5)">
            <rect x="15.5" y="611.5" width="8" height="28" fill="#7efb62" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-4 highcharts-bar-series highcharts-color-undefined " transform="translate(972,395) rotate(90) scale(-1,1) scale(1 1)" width="639" height="385" clip-path="none"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(333,10) scale(1 1)" opacity="1" visibility="visible">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(594,9)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-1 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(333,10) scale(1 1)" opacity="1" visibility="visible">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(12,95)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-2 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(333,10) scale(1 1)" opacity="1" visibility="visible">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(34,181)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-3 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(333,10) scale(1 1)" opacity="1" visibility="visible">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(0,267)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-4 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(333,10) scale(1 1)" opacity="1" visibility="visible">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(28,354)"></g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="318" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:314px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="46" opacity="1">
            <tspan>Occupied Units with Lease Expiration Dates &gt; 7 Days</tspan>
            <tspan dy="14" x="318">in the Past</tspan>
            <title>Occupied Units with Lease Expiration Dates &gt; 7 Days in the Past</title>
        </text>
        <text x="318" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:314px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="123" opacity="1">
            <tspan>Vacant Rented Units with Lease Hold Period &gt; 2</tspan>
            <tspan dy="14" x="318">Months</tspan>
            <title>Vacant Rented Units with Lease Hold Period &gt; 2 Months</title>
        </text>
        <text x="318" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:314px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="207" opacity="1">
            <tspan>Any Units with Lease Terms &gt; 25 Months</tspan>
        </text>
        <text x="318" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:314px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="284" opacity="1">
            <tspan>Occupied, Missing Move-In Date</tspan>
        </text>
        <text x="318" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:314px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="361" opacity="1">
            <tspan>Vacant Unrented Units with Lease Dates</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="333" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">0</text>
        <text x="617" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,000</text>
        <text x="901" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">2,000</text>
        <text x="404" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">250</text>
        <text x="475" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">500</text>
        <text x="546" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">750</text>
        <text x="688" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,250</text>
        <text x="759" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,500</text>
        <text x="830" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,750</text>
        <text x="955.3515625" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">2,250</text>
    </g>
</svg>