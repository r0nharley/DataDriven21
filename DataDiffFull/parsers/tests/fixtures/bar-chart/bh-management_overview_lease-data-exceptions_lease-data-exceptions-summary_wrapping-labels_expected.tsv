Occupied Units with Lease Expiration Dates > 7 Days in the Past	Data	2091.54929577
Vacant Rented Units with Lease Hold Period > 2 Months	Data	42.2535211268
Any Units with Lease Terms > 25 Months	Data	119.718309859
Occupied, Missing Move-In Date	Data	0.0
Vacant Unrented Units with Lease Dates	Data	98.5915492958